include/SFML/Graphics/Font.hpp

```diff
@@ -179,6 +179,21 @@ class SFML_GRAPHICS_API Font
     ////////////////////////////////////////////////////////////
     const Glyph& getGlyph(Uint32 codePoint, unsigned int characterSize, bool bold, float outlineThickness = 0) const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Tell if a codepoint has a corresponding glyph in the font
+    ///
+    /// A font does not have every Unicode character in it. If the glyph
+    /// of a nonexistent codepoint is requested, a font-dependent
+    /// 'missing character' (often a rectangle with a question mark inside)
+    /// is returned.
```

Comment:
How about:
> Most fonts only include a very limited selection of glyphs from specific Unicode subsets, like Latin, Cyrillic, or Asian characters. While code points without representation will return a font specific default character,  it might be useful to verify whether specific code points are included to determine whether a font is suited to display text in a specific language.